images/linux/scripts/installers/aliyun-cli.sh

```diff
@@ -8,7 +8,8 @@
 source $HELPER_SCRIPTS/install.sh
 
 # Install Alibaba Cloud CLI
-downloadUrl=$(get_github_package_download_url "aliyun/aliyun-cli" "contains(\"aliyun-cli-linux\") and endswith(\"amd64.tgz\")")
+# Hardcode the version until 3.0.113 or newer is properly released https://github.com/aliyun/aliyun-cli/issues/414
+downloadUrl=$(get_github_package_download_url "aliyun/aliyun-cli" "contains(\"aliyun-cli-linux\") and endswith(\"amd64.tgz\")" "v3.0.112")
```

Comment:
We can simplify parse logic and use the official source to get the latest version: https://aliyuncli.alicdn.com/aliyun-cli-linux-latest-amd64.tgz